c/tskit/core.c

```diff
@@ -434,6 +438,19 @@ tsk_strerror_internal(int err)
         case TSK_ERR_TOO_MANY_VALUES:
             ret = "Too many values to compress";
             break;
+
+        /* Union errors */
+        case TSK_ERR_LEN_MAP:
```

Comment:
These error codes are a little too vague, as they are quite specific to union. Mabye ``TSK_ERR_UNION_LEN_MAP``, etc?